src/schemas-tests/pa-yaml/v3.0/Examples/Src/Screens/FormsScreen2.pa.yaml

```diff
@@ -46,6 +46,7 @@ Screens:
                       Children:
                         - "First Name_DataCard1":
                             Control: TypedDataCard
+                            IsLocked: true
```

Comment:
nit: can we put IsLocked after the Control and Variant properties?

Reply:
this will keep it consistent with how the object model would serialize it along with keeping type-specific keywords before other optional keywyrs

Reply:
to be clear, move to after Variant, but before Properties and Children